packages/angular/projects/ui-angular/src/lib/components/amplify-sign-in/amplify-sign-in.component.html

```diff
@@ -2,41 +2,64 @@
   Define default contents here
 -->
 <ng-template #signInButton>
-  <button data-amplify-button type="submit">{{ signInButtonText }}</button>
+  <button
+    class="amplify-button"
+    data-fullwidth="true"
+    data-variation="primary"
+    type="submit"
+  >
+    {{ signInButtonText }}
+  </button>
 </ng-template>
 
 <ng-template #signInForm>
-  <amplify-federated-sign-in></amplify-federated-sign-in>
   <form data-amplify-form (submit)="onSubmit($event)" (input)="onInput($event)">
-    <fieldset data-amplify-fieldset [disabled]="isPending">
+    <fieldset
+      class="amplify-flex"
+      style="flex-direction: column"
+      data-amplify-fieldset
+      [disabled]="isPending"
+    >
+      <h3 class="amplify-heading">{{ this.headerText }}</h3>
       <amplify-user-name-alias></amplify-user-name-alias>
       <amplify-form-input
         data-amplify-password
         name="password"
         type="password"
         autocomplete="current-password"
       ></amplify-form-input>
```

Comment:
Are you planning on adding the show password button when you add the tabs?

Reply:
Right, I need to add that in. I'll do that in my sign up PR and create a password-control primitive like you did for the Vue implementation.